includes/formatting.php

```diff
@@ -66,7 +66,17 @@ function get_body_contents( $content ) {
  * @return int The number of replacements made.
  */
 function sideload_images( $post_id ) {
-	$post    = get_post( $post_id );
+	require_once ABSPATH . 'wp-admin/includes/media.php';
+	require_once ABSPATH . 'wp-admin/includes/file.php';
+	require_once ABSPATH . 'wp-admin/includes/image.php';
+
+	$post = get_post( $post_id );
```

Comment:
More a force of habit than anything, but you're right — no sense in loading the dependencies before we've confirmed we need them.